changelog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## version 0.3.1
```

Comment:
Let's call this version 0.4.0, given what it does.

Reply:
Cool I'll do that :)